Commands.md

```diff
@@ -853,3 +854,45 @@ $ git merge-repo git@github.com:tj/git-extras.git master .
 ```
 
 The above merges a remote repo's `master` branch into the current repository's directory, not preserving history.
+
+## git pstash
+
+Similar to builtin `git stash` but keeps your stashes on remote as well.
+
+```bash
+$ git pstash
+$ git pstash save
+```
+
+Any of the above creates new stash with "${USER}_${current_branch_name}_wip" name. Your working directory is left untouched.
```

Comment:
Perhaps this should be `wip/${USER}/${current_branch_name}` to follow branch name convention.